src/queries/common-event-v5.sql

```diff
@@ -59,11 +59,16 @@ FUNCTION `helix-225321.helix_rum.EVENTS_V5`( -- noqa: PRS
       key_bytes = SHA512(domainkey)
       AND (revoke_date IS NULL
         OR revoke_date > CURRENT_DATE(timezone))
-      AND (hostname_prefix = ""
+      AND (
+        hostname_prefix = ""
         OR filterurl LIKE CONCAT("%.", hostname_prefix)
         OR filterurl LIKE CONCAT("%.", hostname_prefix, "/%")
         OR filterurl LIKE CONCAT(hostname_prefix)
-        OR filterurl LIKE CONCAT(hostname_prefix, "/%")))
+        OR filterurl LIKE CONCAT(hostname_prefix, "/%")
+        -- handle comma-separated list of urls, remove spaces and trailing comma
+        OR hostname_prefix IN (SELECT * FROM UNNEST(SPLIT(REGEXP_REPLACE(RTRIM(filterurl, ','), ' ', ''), ',')))
```

Comment:
I would propose to introduce a function for splitting the multiple URLs / hostnames.

Reply:
> I would propose to introduce a function for splitting the multiple URLs / hostnames.

Thank you @lydiapuric, how does this look in terms of naming or other factors?
```
CREATE OR REPLACE FUNCTION helix_rum.URLS_FROM_LIST(inurls STRING)
RETURNS ARRAY<STRING> AS (
  SPLIT(REGEXP_REPLACE(RTRIM(inurls, ','), ' ', ''), ',')
);
```

I created a test version which you can try at:
`SELECT * FROM UNNEST(test_dataset.URLS_FROM_LIST('damian.com,lydia.com, marquise.com,'));`

Reply:
Here's an alternative written as a table function, which do you prefer?

```
CREATE OR REPLACE TABLE FUNCTION helix_rum.URLS_FROM_LIST(inurls STRING) AS (
  SELECT * FROM UNNEST(SPLIT(REGEXP_REPLACE(RTRIM(inurls, ','), ' ', ''), ',')) AS url
);
```

Slightly fewer chars to call it, test at:
`SELECT * FROM test_dataset.URLS_FROM_LIST2('damian.com,lydia.com, marquise.com,');`

Reply:
This function looks good

Reply:
Added table function to PR.